content/docs/csidriver/troubleshooting/powerscale.md

```diff
@@ -16,3 +16,4 @@ Here are some installation failures that might be encountered and how to mitigat
 | If the hostname is mapped to loopback IP in /etc/hosts file, and pods are created using 1.3.0.1 release, after upgrade to driver version 1.4.0 or later there is a possibility of "localhost" as a stale entry in export | Recommended setup: User should not map a hostname to loopback IP in /etc/hosts file |
 | CSI Driver installation fails with the error message "error getting FQDN". | Map IP address of host with its FQDN in /etc/hosts file.     |
 | Driver node pod is in "CrashLoopBackOff" as "Node ID" generated is not with proper FQDN. | This might be due to "dnsPolicy" implemented on the driver node pod which may differ with different networks. <br><br> This parameter is configurable in both helm and Operator installer and the user can try with different "dnsPolicy" according to the environment.|
+| The `kubectl logs isilon-controller-0 -n isilon -c driver` logs shows the driver **Authentication failed. Trying to re-authenticate** when using Session-based authentication | The issue has been resolved from OneFS 9.3, for OneFS versions prior to 9.3 for session-based authentication either smart connect can be created against a single node of Isilon or CSI Driver can be installed/pointed to a particular node of the Isilon else basic authentication can be used by setting isiAuthType in `values.yaml` to 0 |
```

Comment:
This issue has been resolved OneFS 9.3 onwards. OneFS versions prior to 9.3 for session-based authentication either smart connect can be created against a single node of Isilon or CSI Driver can be installed/pointed to a particular node of the Isilon else basic authentication can be used by setting isiAuthType